openvdb_points/tools/AttributeArray.h

```diff
@@ -318,6 +317,12 @@ class TypedAttributeArray: public AttributeArray
     /// Return a new attribute array of the given length @a n with uniform value zero.
     static Ptr create(size_t n);
 
+    /// Down-cast a typeless @a attributeArray to a strongly typed attribute array
```

Comment:
Think we should avoid words like "typeless" and "strongly typed" they can be a bit ambiguous. Let's just say casting an AttributeArray to a TypedAttributeArray.